arcanist/lib/WatchmanTestCase.php

```diff
@@ -139,13 +139,12 @@ function willRunTests() {
   function didRunTests() {
     $this->logTestInfo('didRun');
 
-    foreach ($this->watches as $root => $status) {
-      try {
-        $this->watchmanCommand('watch-del', $root);
-      } catch (Exception $e) {
-        // Swallow
-      }
+    try {
+      $this->watchmanCommand('watch-del-all');
```

Comment:
it would be great to have a more proactive/positive assertion that this is functioning; could you add a new test in `tests/integration` that watches a couple of dirs then asserts that all of those were deleted?